### PR TITLE
feat: add token to sent mails of koala farm

### PR DESCRIPTION
**Pull request by mweibel**

### CODE REVIEW

File: src/Classes/Product/Product.php [hunk 1]
```diff
@@ -2,12 +2,16 @@
...
 
+use Helio\Panel\Model\User;
+
 interface Product
 {
     public function baseURL(): string;
 
     public function confirmURL(): string;
 
+    public function URL(User $user): string;
```
Comment (cbuchli) [inline src/Classes/Product/Product.php]: It's pretty straight forward but one thing: Could we find a better name than "URL" for this?

Comment (mweibel) [reply src/Classes/Product/Product.php]: 😄 `link`?

Comment (mweibel) [reply src/Classes/Product/Product.php]: doesn't feel much better tho

Comment (mweibel) [reply src/Classes/Product/Product.php]: URL is consistent with baseURL/confirmURL, that's why I chose but I'm also not that happy about it

Comment (cbuchli) [reply src/Classes/Product/Product.php]: What's it actually used for? maybe "directLoginURL"? or something like that... 🤔

Comment (mweibel) [reply src/Classes/Product/Product.php]: it's the call to action link URL 🤔

Comment (mweibel) [reply src/Classes/Product/Product.php]: named it `callToActionURL` now

**Pull request merged**